evalbase/evalbase/models.py

```diff
@@ -38,7 +38,7 @@ class Conference(models.Model):
     admin_contact = models.EmailField()
     complete = models.BooleanField()
     agreements = models.ManyToManyField('Agreement', blank=True)
-    results_root = models.CharField(
+    results_root: str = models.CharField(
```

Comment:
I don't want to add type annotations unless the whole codebase gets the treatment.  I also haven't coded in Python with type annotations before so this might be a big change to do.

Suggest dropping this change.